1.6/contrib/configuration/jobs/OpenShift Sample/config.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<project>
+  <actions/>
+  <description></description>
+  <keepDependencies>false</keepDependencies>
+  <properties>
+    <hudson.model.ParametersDefinitionProperty>
+      <parameterDefinitions>
+
+        <hudson.model.TextParameterDefinition>
+          <name>OPENSHIFT_API_URL</name>
+          <description>URL of the OpenShift api endpoint.</description>
```

Comment:
indicate this can be left blank to run against the API server for the cluster Jenkins is running in?